CityOfDetroit/bloom
# fix: remove redis dependency

# Pull Request Template

## Issue Overview

This PR addresses #1619 


## Description

Removes all redis and redis-related dependencies. 

Redis is currently only being used by one service (via bull), the listing-notifications.ts. However, this was just a mocked out service that wasn't being called anywhere. There were plans at one point to auto send emails when listings go live but was never built

## How Can This Be Tested/Reviewed?

Remove `REDIS_TLS_URL`, `REDIS_URL`, and `REDIS_USE_TLS` from your .env file for backend/core.

The backend should start up correctly 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
